src/remotes/mod.rs

```diff
@@ -65,8 +66,13 @@ fn get_api_key(domain: &str) -> String {
         Some(key) => key,
         None => {
             let mut newkey = String::new();
-            println!("No API token for {} found. See https://github.com/arusahni/git-req/wiki/API-Keys for instructions.", domain);
-            print!("{} API token: ", domain);
+            println!(
+                "{}",
+                format!("No API token for {} found. See https://github.com/arusahni/git-req/wiki/API-Keys for instructions.", domain).red()
+                );
+            print!("{}",
+                   format!("{} API token: ", domain).red()
```

Comment:
Since this is a prompt, let's leave it uncolored.

Reply:
I'm not quite sure here. Should I uncolor both - 71 and 74 lines or only 74?

Reply:
You're right - both are of single prompt. Will remove color from them both to be consistent. Sorry, never mind my prev.comment 🙂